tests/unit/test_media_backfill_unit.py

```
@@ -0,0 +1,168 @@
+import json
+import os
+import sys
+import types
+import unittest
+from contextlib import redirect_stdout
+from unittest.mock import Mock, patch
+
+
+class MediaBackfillUnitTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.9)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)